content/actions/automating-builds-and-tests/building-and-testing-ruby.md

```diff
@@ -54,9 +54,12 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Ruby
-        uses: ruby/setup-ruby@477b21f02be01bcb8030d50f37cfec92bfa615b6
+      # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+      # change this to (see https://github.com/ruby/setup-ruby#versioning):
+      # uses: ruby/setup-ruby@v1
```

Comment:
I really like this idea. I'm working with the product team on the wording (since we recommend using the SHA for security reasons).

I don't want to hold up your PR while I wait on a response. For now, I am going to remove this. Once the wording is settled, we'll add a similar comment to this and other examples.

```suggestion
```

Reply:
This is the wording used in starter workflows, and so already officially accepted, see https://github.com/actions/starter-workflows/blob/main/ci/ruby.yml
@dentarg Would you mind making a new PR to restore this?

It's important, and this might help regarding issues like https://github.com/ruby/setup-ruby/issues/319#issuecomment-1108590210

Reply:
I'll wait a while to see if there will be any response to https://github.com/github/docs/pull/15240#issuecomment-1108589073